frontend/packages/ux-editor/src/containers/ComponentPreview/RadioGroupPreview/RadioGroupPreview.tsx

```diff
@@ -25,9 +26,14 @@ export const RadioGroupPreview = ({
 }: RadioGroupPreviewProps) => {
   const t = useText();
   const tRadios = (key: string) => t(`ux_editor.radios_${key}` as TranslationKey);
-
   const radioGroupName = useRef(generateRandomId(12));
 
+  const { isValid, error } = validateComponent(component);
+
+  if (!isValid) {
+    return <ErrorMessage>{tRadios(`error_${error}`)}</ErrorMessage>;
```

Comment:
I think I would prefer the error message to be shown in the edit modal, rather than blocking the entire component.
Could we at least show it together with the other parts of the component (the ones that are still valid)? Or do the validation `onSave`?

Reply:
I agree, just didn't prioritize it since we are going to redesign this part anyways. I'll have a look at it!